draft-ietf-tls-tls13.md

```diff
@@ -2159,7 +2176,15 @@ Although X9.62 supports multiple point formats, any given curve
 MUST specify only a single point format. All curves currently
 specified in this document MUST only be used with the uncompressed
 point format (the format for all ECDH functions is considered
-uncompressed).
+uncompressed). Peers MUST validate each other's public value Y by ensuring
+that the point is a valid point on the elliptic
+curve. For all elliptic curves except for x25519 and x448, the appropriate
```

Comment:
I am fine with specifying the details for X25519 and X448. But I really need to research it before we include anything. In particular, there are some bad points in Curve25519 that may have implications for things like Triple-Handshake style attacks. I don't know if these cases are already handled by the X25519 specification, so I'll try to figure it out.